### PR TITLE
Create eip155-87.json

### PR DESCRIPTION
**Pull request by nova-network-inc**
Add Nova Network

### CODE REVIEW

**Review by alirezaayande — APPROVED** [review]
_data/chains/eip155-87.json

**Review by alirezaayande — APPROVED** [review]
nova-network-inc:patch-4

**Comment by nova-network-inc** [comment]
Can you please advise on what the problem is? I'm sorry but I'm not being able to figure it out.

File: _data/chains/eip155-87.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "symbol": "NBX",
+    "decimals": 18
+  },
+  "infoURL": "https://novanetwork.io/",
+  "shortName": "nbx",
+  "chainId": 87,
+  "networkId": 87,
+  "explorers": [{
+    "name": "novanetwork",
+    "url": "https://explorer.novanetwork.io/",
```
Comment (ligi) [inline _data/chains/eip155-87.json:21]:
```suggestion
    "url": "https://explorer.novanetwork.io",
```

**Comment by ligi** [comment]
The explorer does not load for me - so I cannot check if it is EIP compliant

**Comment by nova-network-inc** [comment]
Just access without the https (eg. http://explorer.novanetwork.io/) and it will work. It is EIP compliant.

**Comment by nova-network-inc** [comment]
Do you know why this one isn't building, and the testnet (107) is? Am I doing something wrong here?

File: _data/chains/eip155-87.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Nebula X",
+    "symbol": "NBX",
+    "decimals": 18
+  },
+  "infoURL": "https://novanetwork.io/",
+  "shortName": "nbx",
+  "chainId": 87,
+  "networkId": 87,
```
Comment (ligi) [inline _data/chains/eip155-87.json:18]:
```suggestion
  "networkId": 87
```

**Pull request merged**